generic3g/tests/Test_InfoUtilities.F90

```diff
@@ -109,4 +105,46 @@ subroutine test_setPrivate_is_private()
 
    end subroutine test_setPrivate_is_private
 
+   @test
+   subroutine test_setInternal()
+      type(ESMF_State) :: state
+      type(ESMF_Field) :: field
+      integer :: status
+      real(ESMF_KIND_R4), allocatable :: w(:)
+
+      state = ESMF_StateCreate(name='import', _RC)
+      field = ESMF_FieldEmptyCreate(name='f', _RC)
+      call ESMF_StateAdd(state, [field], _RC)
+
+      call MAPL_InfoSetInternal(state, short_name='f', key='a', values=[1., 2.], _RC)
+      call MAPL_InfoGetInternal(state, short_name='f', key='a', values=w, _RC)
+
+      @assert_that(w, is(equal_to([1.,2.])))
+
```

Comment:
```suggestion
      real(ESMF_KIND_R4), allocatable :: w(:)
      real(ESMF_KIND_R4), parameter = expected(2) = [1., 2.]

      state = ESMF_StateCreate(name='import', _RC)
      field = ESMF_FieldEmptyCreate(name='f', _RC)
      call ESMF_StateAdd(state, [field], _RC)

      call MAPL_InfoSetInternal(state, short_name='f', key='a', values=expected, _RC)
      call MAPL_InfoGetInternal(state, short_name='f', key='a', values=w, _RC)

      @assert_that(w, is(equal_to(expected)))

```

Reply:
It helps to make the expected value a parameter rather than a literal.

Reply:
Agreed.  I was lazy. Thank you.

Reply:
Done in sandbox.